t/22_ipaddr2.t

```diff
@@ -355,6 +355,40 @@ subtest '[ipaddr2_deployment_logs]' => sub {
     ok(($called eq 1), "az_vm_diagnostic_log_get called");
 };
 
+
+subtest '[ipaddr2_cluster_sanity]' => sub {
+    my $ipaddr2 = Test::MockModule->new('sles4sap::ipaddr2', no_auto => 1);
+    $ipaddr2->redefine(ipaddr2_bastion_pubip => sub { return '1.2.3.4'; });
+    my @calls;
+
+    $ipaddr2->redefine(ipaddr2_ssh_internal => sub {
+            my (%args) = @_;
+            push @calls, ["VM$args{id}", $args{cmd}, ""];
+            return;
+    });
+    $ipaddr2->redefine(ipaddr2_ssh_internal_output => sub {
+            my (%args) = @_;
+            my $out;
+            # return only good vibes
+            if ($args{cmd} =~ /crm status/) {
+                $out = "Some generic string that is good for cluster_status_matches_regex";
+            } elsif ($args{cmd} =~ /crm configure show/) {
+                $out = "primitive primitive primitive that is what the ipaddr2_cluster_sanity wants";
+            } else {
+                $out = "Galileo Galilei";
```

Comment:
On a scale 1 to 10. How random is that ? :D

Reply:
well, as most of the UT there's a theme per file. Can you guess the theme here? :it: :brain: